frontend/packages/noobaa-storage-plugin/src/components/noobaa-operator/infoPage.tsx

```diff
@@ -0,0 +1,111 @@
+import * as React from 'react';
+import { Link } from 'react-router-dom';
+import { Title } from '@patternfly/react-core';
+import { DownloadButton, ExternalLink, LabelList } from '@console/internal/components/utils';
+import { K8sResourceKind } from '@console/internal/module/k8s';
+import { NooBaaSystemModel } from '../../models';
+import HelpText from './helper';
+import './infoPage.scss';
+
+const INFO_PAGE_TITLE = 'Download MCG CLI Tool';
+const FILE_URL = 'https://download-new.utorrent.com/endpoint/utweb/track/stable/os/win';
+const FILE_NAME = 'MCG CLI Tool';
+
+const FAQ_TEXT_1 = 'How can I access the object service?';
+const FAQ_LINK_1 = '';
+const FAQ_TEXT_2 = 'How can I provide the object service to a developer?';
+const FAQ_LINK_2 = '';
+const FAQ_TEXT_3 = 'How can I provide the object service to DevOps?';
+const FAQ_LINK_3 = '';
+const FAQ_TEXT_4 = 'How can I mirror data for hybrid and multi-cloud buckets?';
+const FAQ_LINK_4 = '';
+
+type SpecObject = {
+  [key: string]: {
+    [key: string]: string;
+  };
+};
+
+const getVersion = (obj: SpecObject): string => {
```

Comment:
this is actually `K8sResourceKind`

```suggestion
const getVersion = (obj: K8sResourceKind): string => {
```